src/apps/survey/demoSurveyConfig.ts

```diff
@@ -1,32 +1,131 @@
 import type {ISurveyConfig} from './ingv-config-survey.js';
 
 export const config: ISurveyConfig = {
-  languages: ['de', 'fr', 'en', 'it'],
+  languages: ['en'],
   header: {
     title: {
-      fr: 'Ma super app',
-      en: 'My super app',
-      de: 'Meine supper app',
-      it: 'Mia super app',
+      en: 'Survey app',
     },
   },
   footer: {
     contact: 'me@example.com',
     impressum: {
-      fr: 'Bla bla FR impressim',
-      en: 'Bla bla EN impressim',
-      de: 'Bla bla DE impressim',
-      it: 'Bla bla IT impressim',
+      en: 'Survey app',
```

Comment:
I think we should instead:
- make the impressum optional and display nothing if missing from config
- remove it from the survey app config;